Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -57,21 +57,20 @@ RewardCellViewModelOutputs {
     }
     let projectAndReward = Signal.zip(project, reward)
 
-    self.conversionLabelHidden = project.map {
-      !needsConversion(projectCountry: $0.country, userCountry: AppEnvironment.current.config?.countryCode)
-    }
-
+    self.conversionLabelHidden = project.map(needsConversion(project:) >>> negate)
```

Comment:
This is the meat of the new logic! We generalize the old and new paths (with or without the API returning `current_currency` to a `needsConversion(project:)`.

Cleaned things up by composing this against `negate`, and composing it in the filter below against `first`.